File: string/string8.cpp
#include <iostream>
#include <string>

std::string input = "python count"; 


int main(){
    std::cout << input.size();
    int count = 0;
    for(int i = 0; i<input.size(); i++){
        if(input[i] == ' '){
            count++;
        }
        
    }
    count = count + 1;
    std::cout << count << std::endl;
}

File: string/string8.go
package main

import (
	"fmt"
)

var input string = "python count"; 
var count int = 0;

func main(){
	for i := 0; i<len(input); i++ {
		if input[i] == ' '{
			count = count + 1;
		}
	}
	count = count + 1;
	fmt.Printf("%d \n", count);
}